site/app/templates/submission/homework/SubmitBox.twig

```diff
@@ -187,15 +187,23 @@
 
                     {% for image in n.images %}
                         <img src="{{ attribute(image_data, image.name) }}"
+
                         {% if image.width > 0 %}
                             width="{{ image.width }}"
                         {% endif %}
+
                         {% if image.height > 0 %}
                             height="{{ image.height }}"
                         {% endif %}
+
+                        {% if image.alt is defined %}
+                            alt="{{ image.alt }}"
+                        {% endif %}
```

Comment:
I think I'd prefer not having the foreach to apply the alt attribute below and just do:
```
alt="{{ image.alt | default('Instructor Provided Image') }}"
```
here.

Reply:
I believe this code will be refactored as part of #3728 (we won't have multiple images).
So I'm going to merge this.  But I'll have Kevin look at your feedback.